### PR TITLE
Release v0.8

### PR DESCRIPTION
**Pull request by jm-clius**
Draft release notes for release `v0.8`: https://github.com/status-im/nim-waku/issues/870

Let me know if I skipped anything.

### CODE REVIEW

**Review by kaiserd — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,30 +1,54 @@
...
+- Support for ambient peer discovery using [`33/WAKU-DISCV5`](https://github.com/vacp2p/rfc/pull/487)
 
 ### Changes
 
-- A new type `Timestamp` for all timestamps is introduced (currently an alias for int64).
-- All timestamps now have nanosecond resolution.
-- `waku-store` protocol identifier is updated to `/vac/waku/store/2.0.0-beta4`
+- Metrics: now monitoring content topics and the sources of new connections
+- Metrics: improved default fleet monitoring dashboard
+- Introduced a `Timestamp` type (currently an alias for int64).
```
Comment (s1fr0) [inline CHANGELOG.md:24]: Should we mention the change in protobuffer timestamp field to 10?

**Review by staheri14 — APPROVED** [review]
LGTM!

File: CHANGELOG.md (hunk 1 above)
Comment (jm-clius) [reply CHANGELOG.md:24]: Ah, sorry! Saw this comment and forgot to respond. Added to [the release notes](https://github.com/status-im/nim-waku/releases/tag/v0.8) and will update in CHANGELOG in subsequent PR.

**Pull request merged**